airconsole-1.9.0.js

```diff
@@ -1126,6 +1127,27 @@ AirConsole.prototype.onResume = function() {};
  *                   ONLY PRIVATE FUNCTIONS BELLOW                           *
  * ------------------------------------------------------------------------- */
 
+/**
+ * Determines if AirConsole by default should have player silencing enabled or not.
+ * @returns If the player should be silenced based on environment factors.
+ * @private
+ */
+AirConsole.prototype.getDefaultPlayerSilencing_ = function() {
```

Comment:
Give it a go on 61 after release please

Reply:
Checked and fixed now ...
Named captures were indeed 64+ only. Thank you for the push back.